app/services/proofing/resolution_result_adjudicator.rb

```diff
@@ -34,8 +41,25 @@ def should_proof_state_id?
 
     private
 
+    def errors
+      resolution_result.errors.
+        merge(state_id_result.errors).
+        merge(device_profiling_result.errors || {})
+    end
+
+    def exception
+      resolution_result.exception || state_id_result.exception || device_profiling_result.exception
+    end
+
+    def timed_out?
+      resolution_result.timed_out? || state_id_result.timed_out? ||
+        device_profiling_result.timed_out?
+    end
+
     def result_and_adjudication_reason
-      if resolution_result.success? && state_id_result.success?
+      if device_profiling_result.exception?
+        [false, :device_profiling_exception]
```

Comment:
do we maybe want to add multiple reasons? like:
```
[true/false, resolution_reason, device_profiling_reason]
```

or maybe we have two result methods, and then overall success is an `&&` between them?

```ruby
def resolution_result_and_reason
  [true, :pass_resolution_and_state_id]
  # ...
end

def device_profiling_result_and_reason
  [false, :device_profiling_exception]
end

resolution_result, resolution_reason = resolution_result_and_reason
device_profiling_result, device_profiling_reason = device_profiling_result_and_reason

result = {
  success: resolution_result && device_profiling_result,
  resolution_reason:,
  device_profiling_reason:
}

```

because I feel like it would still be valuable to see what the resolution decision would have been without the TMX exception